src/hotspot/share/opto/c2_globals.hpp

```diff
@@ -785,7 +785,8 @@
           range(0, max_juint)                                               \
                                                                             \
   product(bool, UseProfiledLoopPredicate, true,                             \
-          "Move predicates out of loops based on profiling data")           \
+          "Move predicates out of loops based on profiling data. "          \
+          "Requires UseLoopPredicate to be turned on (default).")           \
```

Comment:
It was already a bit vague before but I suggest to be more precise that we move checks with an uncommon trap out of a loop (and the resulting check before the loop is then a predicate):
```
Move checks with an uncommon trap out of loops based on profiling data. Requires [...]
```